packages/chip-input/src/ux-chip-input.ts

```diff
@@ -181,4 +186,76 @@ export class UxChipInput implements UxComponent {
 
     this.styleEngine.applyTheme(newValue, this.element);
   }
+
+  public focusedChanged(focused: boolean) {
+    if (focused === true) {
+      this.element.classList.add('ux-chip-input--focused');
+    } else {
+      this.element.classList.remove('ux-chip-input--focused');
+    }
+
+    this.element.dispatchEvent(DOM.createCustomEvent(focused ? 'focus' : 'blur', { bubbles: false }));
+  }
+
+  public focus() {
+    this.textbox.focus();
+  }
+
+  public blur() {
+    if (document.activeElement === this.textbox) {
+      this.textbox.blur();
+      this.addChip();
```

Comment:
adding chip on blur is quite a surprise, do we want this? at least it needs to be explicitly configured

Reply:
took this behavior from the previous blur listener (see detached() method on previous commit). For having used this component already in several projects I must say that I find it very practical from an end-user point of view.

When you say that it must be explicitly configured, do you think about a bindable to configure this as a component instance scope or from the plugin configure api to set it for all ux-chip-input in a project ?

Reply:
I see. yes it should be a bindable where we can toggle this behavior. Thanks for pointing out the existing behavior

Reply:
Will do.

As I like the `addChipOnBlur` behavior I would go for a default to `true` and allow user to opt-out. However, from your original remark you might suggest to have it `false` as default and require user to explicitly opt-in for it. Your thoughts ?

Reply:
Well, ux is used in prod, so we cannot flip it around. maybe default to `true` and have option to `false` it